src/components/AlertBanner/AlertBanner.tsx

```diff
@@ -1,24 +1,52 @@
-import React, { forwardRef, useMemo } from "react";
+import React, { ForwardedRef, forwardRef, ReactElement, useMemo } from "react";
 import cx from "classnames";
-import PropTypes from "prop-types";
 import Button from "../../components/Button/Button";
 import Icon from "../../components/Icon/Icon";
 import CloseSmall from "../../components/Icon/Icons/components/CloseSmall";
-import { BACKGROUND_COLORS } from "./AlertBannerConstants";
+import { ALERT_BANNER_BACKGROUND_COLORS } from "./AlertBannerConstants";
+import { NOOP } from "../../utils/function-utils";
+import VibeComponentProps from "../../types/VibeComponentProps";
+import { AlertBannerLinkProps } from "./AlertBannerLink/AlertBannerLink";
+import { AlertBannerButtonProps } from "./AlertBannerButton/AlertBannerButton";
+import { AlertBannerTextProps } from "./AlertBannerText/AlertBannerText";
 import "./AlertBanner.scss";
 
-const NOOP = () => {};
+interface AlertBannerProps extends VibeComponentProps {
+  /**
+   * Set external styling to the progress bar.
+   */
+  className?: string;
+  backgroundColor?: ALERT_BANNER_BACKGROUND_COLORS;
```

Comment:
ALERT_BANNER_BACKGROUND_COLORS should be enum - lets change the name of it to the enum to camel case structure